pywwise/structs.py

```diff
@@ -359,3 +360,379 @@ class SoundBankGenerationInfo:
+	EAudioObjectOptions instead."""
+	
+	def __hash__(self):
+		""":return: The AudioObject hash."""
+		return hash(self.audio_object_id)
+
+
+@_dataclass
+class BusPipelineInfo:
+	"""Contains information about an audio bus captured in the profiler"""
+	
+	pipeline_ID: int
+	"""Pipeline ID of the bus. Unsigned Integer 32-bit. Range: [0,4294967295]"""
+	
+	gameObject_ID: int
+	"""Game Object ID corresponding to the voice. Unsigned Integer 64-bit. Range: [0,18446744073709551615]"""
+	
+	object_GUID: GUID
+	"""Object GUID corresponding to the bus. An object GUID of the form: {aabbcc00-1122-3344-5566-77889900aabb}."""
+	
+	other: dict[EBusOptions | str, _Any] = _field(default_factory=dict)
+	
+	def __hash__(self):
+		""":return: The BusPipelineInfo hash."""
+		return hash(self.pipeline_ID)
+
+
+@_dataclass
+class CPUStatisticsInfo:
+	"""Information about the amount of CPU percentage used by each element."""
+	
+	element_Name: str
+	"""The name of the element on which we calculate CPU usage."""
+	
+	id: int
+	"""Class ID of the element."""
+	
+	instances: int
+	"""An estimation of the number of instances of the element."""
+	
+	type: str
+	"""The type of element. For example, Codec, Source, Effect, Mixer or Sink."""
+	
+	percent_Inclusive: float
+	"""The percentage of CPU time spent in the execution of the element and those that it uses (calls)."""
+	
+	percent_Exclusive: float
+	"""The percentage of CPU time spent only in the execution of the element itself."""
+	
+	milliseconds_Inclusive: float
+	"""The milliseconds of CPU time spent in the execution of the element and those that it uses (calls)."""
+	
+	milliseconds_Exclusive: float
+	"""The milliseconds of CPU time spent only in the execution of the element itself."""
+
+
+@_dataclass
+class GameObjectRegistrationData:
+	"""Data of a profiled game object and its registration data."""
+	
+	id: int
+	"""The ID of the game object. Unsigned Integer 64-bit. Range: [0,18446744073709551615]"""
+	
+	name: str
+	"""The name of the game object."""
+	
+	registration_Time: int
+	"""The time at which the game object was registered. Integer 32-bit. Range: [-2147483648,2147483647]"""
+	
+	unregistration_Time: int
+	"""The time at which the game object was unregistered. Integer 32-bit. Range: [-2147483648,2147483647]"""
+
+
+@_dataclass
+class LoadedMediaInfo:
+	"""Information about a media file loaded into memory as a result of the PrepareEvent() and PrepareGameSyncs()
+	functions."""
+
+	media_Id: ShortID
+	"""The short ID of the media file"""
+	
+	file_Name: Name
+	"""The name of the media file."""
+	
+	format: str
+	"""The audio format of the media file."""
+	
+	size: int
+	"""The size (in bytes) of the media file."""
+	
+	sound_Bank: Name
+	"""The name of the SoundBank that contains the media file."""
+
+
+@_dataclass
+class PerformanceMonitorCounterInfo:
+	"""Information about a performance monitor counter and its value"""
+	
+	name: Name
```

Comment:
This can be a `str`. `Name` is more meant for working with Wwise Objects (containers, sounds, sources, etc). But if you did have to use `Name` here because of some type-check or cast, that's totally fine – in that case, you can keep it.